### PR TITLE
Trigger archiver checker job monday 10pm weekly, add monitoring of bu…

### PR DESCRIPTION
**Pull request by duttonw**
…lk and default queues, as original is all queues

### CODE REVIEW

File: templates/default/ckan-monitor-job-queue.sh.erb [hunk 1]
```diff
@@ -13,3 +14,33 @@ else
     MAX_AGE=0
 fi
 aws cloudwatch put-metric-data --region <%= node['datashades']['region'] %> --namespace CKAN --metric-data "MetricName=job_queue_size,Dimensions=[{Name=Application,Value=<%= node['datashades']['app_id'] %>},{Name=Environment,Value=<%= node['datashades']['version'] %>}],Value=$JOB_COUNT" "MetricName=max_job_age,Dimensions=[{Name=Application,Value=<%= node['datashades']['app_id'] %>},{Name=Environment,Value=<%= node['datashades']['version'] %>}],Value=$MAX_AGE"
+
+# Default queue
```
Comment (ThrawnCA) [inline templates/default/ckan-monitor-job-queue.sh.erb:18]: This should probably be extracted to a function. Also, which queue will the weekly archiver run use? Perhaps we should monitor the 'priority' queue.

File: recipes/ckanweb-deploy-exts.rb [hunk 2]
```diff
@@ -292,6 +292,19 @@
...
 			end
+
+			template "/usr/local/bin/archiverTriggerAll.sh" do
+				source 'archiverTriggerAll.sh'
+				owner 'root'
+				group 'root'
+				mode '0755'
+			end
+
+			#Trigger at 10pm monday nights weekly
```
Comment (ThrawnCA) [inline recipes/ckanweb-deploy-exts.rb:303]: Wouldn't the weekend be better?

Comment (duttonw) [reply recipes/ckanweb-deploy-exts.rb:303]: is after peak hour and url checks should be done by early morning. Another reason is do you want to be working on the weekend if it breaks?

File: templates/default/ckan-monitor-job-queue.sh.erb (hunk 1 above)
Comment (duttonw) [reply templates/default/ckan-monitor-job-queue.sh.erb:18]: i left out one of the queues as you can deduce it from the others

Comment (duttonw) [reply templates/default/ckan-monitor-job-queue.sh.erb:18]: weekly archiver uses the bulk queue, priority is only used by resource updates/package updates

Comment (ThrawnCA) [reply templates/default/ckan-monitor-job-queue.sh.erb]: Fair enough. I did the refactor, though, so adding priority monitoring was only a one-liner.

**Pull request merged**